docs/static/upgrading.asciidoc

```diff
@@ -111,26 +111,23 @@ Before upgrading Logstash:
 +
 There you can find info on these topics and more:
 
-** <<bc-ecs-compatibility,Elastic Common Schema (ECS) compatibility on by default>>
-** <<security-on-8.0,Secure communication with Elasticsearch>>
-** <<bc-field-ref-parser,Field parser is more strict>>
-
+** <<ssl-settings-9.0,Changes to SSL settings in {ls} plugins>>
  
 If you are installing Logstash with other components in the Elastic Stack, also see the
 {stack-ref}/index.html[Elastic Stack installation and upgrade documentation].
 
-NOTE: Upgrading between non-consecutive major versions (6.x to 8.x, for example) is not supported. 
-We recommend that you upgrade to {prev-major-last}, and then upgrade to 8.0.
+NOTE: Upgrading between non-consecutive major versions (7.x to 9.x, for example) is not supported.
+We recommend that you upgrade to {prev-major-last}, and then upgrade to 9.0.
 
 [discrete]
 [[upgrade-to-previous]]
-==== Upgrade to {ls} {prev-major-last} before upgrading to 8.0
+==== Upgrade to {ls} {prev-major-last} before upgrading to 9.0
 
-If you haven't already, upgrade to version {prev-major-last} before you upgrade to 8.0. If
+If you haven't already, upgrade to version {prev-major-last} before you upgrade to 9.0. If
```

Comment:
`{prev-major-last}` magically 🎆 showing `8.17.1` disappeared (I am guessing Karen _quickly_ fixed it or something was wrong when I hit accessed the page) and now rendered correctly!